h/static/styles/help-page.scss

```diff
@@ -1,10 +1,10 @@
 $base-font-size: 16px;
 $base-line-height: 22px;
 
-@import './reset';
-@import './common';
-@import './elements';
-@import './mixins/icons';
+@import 'reset';
```

Comment:
It's more a conversation for beer than a pull request, but here's the thing: there are two different kinds of `@import` in SASS:

1. Those that result in output.
2. Those that do not (i.e. mixin modules).

If you use imports in SASS the same way you're using using imports in other code (i.e. assume they are side-effect free and import them as close as possible to where they're used) then sooner or later you're going to mistake an import of type